block/block_service.proto

```diff
@@ -220,45 +220,45 @@ message PublishStreamResponse {
          */
         uint64 block_number = 2;
     }
+}
 
+/**
+ * An enumeration indicating the status of this request.
+ *
+ * This enumeration describes the reason a block stream
+ * (sent via `writeBlockStream`) ended.
+ */
+enum EndOfStreamResponseCode {
```

Comment:
Probably best if we name this more like the other response codes in this file.
`PublishResponseCode` or `PublishStreamResponseCode` should work.